app/jenkins-for-jira-ui/src/__mocks__/getAllJenkinsServers.ts

```diff
@@ -0,0 +1 @@
+export const getAllJenkinsServers = jest.fn(() => Promise.resolve([]));
```

Comment:
/nit /opinion-piece
`export const getAllJenkinsServers = jest.fn(async () => []);`
I like this way, async will implicitly return a promise so you can simplify a smidge 🤷